modules/pipeline/services/pipelinesvc/create_v2.go

```diff
@@ -50,10 +50,11 @@ func (s *PipelineSvc) CreateV2(req *apistructs.PipelineCreateRequestV2) (*spec.P
 	// 立即执行一次
 	if req.AutoRunAtOnce {
 		_p, err := s.RunPipeline(&apistructs.PipelineRunRequest{
-			PipelineID:        p.ID,
-			ForceRun:          req.ForceRun,
-			IdentityInfo:      req.IdentityInfo,
-			PipelineRunParams: req.RunParams,
+			PipelineID:             p.ID,
+			ForceRun:               req.ForceRun,
+			IdentityInfo:           req.IdentityInfo,
+			PipelineRunParams:      req.RunParams,
+			ConfigManageNamespaces: req.ConfigManageNamespaces,
```

Comment:
No need this line. It's duplicated with create.

Reply:
done